src/diagnostics/MOM_wave_speed.F90

```diff
@@ -704,7 +709,7 @@ subroutine wave_speeds(h, tv, G, GV, US, nmodes, cn, CS, w_struct, u_struct, u_s
     Sc, &         ! A column of layer salinities after convective instabilities are removed [S ~> ppt]
     Rc, &         ! A column of layer densities after convective instabilities are removed [R ~> kg m-3]
     Hc_H          ! Hc(:) rescaled from Z to thickness units [H ~> m or kg m-2]
-  real :: I_Htot  ! The inverse of the total filtered thicknesses [Z ~> m]
+  real :: I_Htot  ! The inverse of the total filtered thicknesses [Z-1 ~> m-1]
```

Comment:
or is it `H-1` ?

Reply:
The units of I_Htot will become [H-1] in the next PR, but that one is not spread across so many files.